default: warning
logs:
  requests.packages.urllib3.connectionpool: error
  py.warnings: error

